src/transformer.js

```diff
@@ -44,7 +65,7 @@ const processSync = (source, filename, jestOptions) => {
   const preprocessor = require.resolve('./preprocess.js')
 
   const preprocessResult = execSync(
-        `node --unhandled-rejections=strict --abort-on-uncaught-exception "${preprocessor}"`,
+        `node --experimental-vm-modules --unhandled-rejections=strict --abort-on-uncaught-exception "${preprocessor}"`,
```

Comment:
we don't need to add that flag here since we know we're in CJS mode, right?
```suggestion
        `node --unhandled-rejections=strict --abort-on-uncaught-exception "${preprocessor}"`,
```

Reply:
In theory that should be true. Let's try :)